docs/orchestration/server/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+# Troubleshooting
+
+Having trouble with Prefect Server? This page contains solutions to common problems.
+
+## How to know if there's a problem
+
+After running `prefect server start`, you should be able to view the Prefect UI in a web browser at the address [http://localhost:8080](http://localhost:8080). This can work even if the server hasn't fully started yet.
+
+You'll know there's a problem if the color of the connection menu in the upper-right-corner Prefect UI transitions between yellow and red, and never turns green.
+
+You will see the message `Couldn't connect to Prefect Server at http://localhost:4200/graphql` when you click on the connection menu.
+
+![](/orchestration/server/could-not-connect.png)
```

Comment:
Looks like a misspelling here with docs/.vuepress/public/orchestration/server/coult-not-connect.png

Reply:
Thanks -- and thanks for your copy suggestions! I'm not sure how I should link to the images, mostly because I don't know exactly how to build the docs and check my work. I did try to follow the path + URLs used elsewhere, e.g.: https://github.com/PrefectHQ/prefect/blob/0f3270f0de4418bd089b8e9bd65da42f5a39ed86/docs/core/about_prefect/thinking-prefectly.md?plain=1#L199

Reply:
It looks to me like you're linking correctly, but I'm setting up to review a local doc build after a couple meetings.
You can follow the directions here to preview locally, with the caveat that the build process works best (in my short experience) with Node 14 LTS, _not_ Node 16. https://docs.prefect.io/core/development/documentation.html#previewing-docs-locally
I'm using nvm to make sure the correct version is set when working on the docs.

Reply:
Thanks for that tip! I was just trying to troubleshoot the docs build with node 17. I'll downgrade to 14.

Reply:
Let us know how that works. I am trying to capture doc build troubleshooting notes as I become more familiar with it.

Reply:
It worked! Glad I checked locally because doing so revealed that I needed to add the new page to the `sidebar` object in `docs/.vuepress/config.js`.

Also, now I see what you mean about the filename. 😅 Fixed!

Reply:
Confirmed this builds correctly and everything looks good. Thanks @abrookins